examples/films.js

```diff
@@ -29,6 +29,8 @@ const api = joiql({
 // Koa 2 style middleware to resolve the request
 // (using promises in anticipation of async/await)
 api.use((ctx, next) => {
+  ctx.res.twb = { title: 'There will be blood' }
+  ctx.res.her = { title: 'Her' }
   ctx.res.film = { title: 'Paul Blart Mall Cop' }
```

Comment:
As you can see here we can fulfill aliases in the response by adding to them like we do non-aliases.

This screenshot might better illustrate that:

![image](https://cloud.githubusercontent.com/assets/555859/20739436/8c84fe78-b688-11e6-8bcc-ddb5dc791863.png)